src/index.js

```diff
@@ -18,16 +20,34 @@ let tasks = [
   },
 ];
 
-// document.querySelector('resync-icon').src = Icon;
+// document.querySelector('.resync-icon').src = ResyncIcon;
+// document.querySelectorAll('.option-icon').src = OptionIcon;
 
-window.onload = ()=>{
+window.onload = () => {
   // fetch tasks
+  tasks.forEach((task) => {
+    document.querySelector('.listTodo').innerHTML += `<article class="todo-article">
+    <div>
+      <input type="checkbox" name="check" id="checkbox-${task.index}" value="">
+      <label for="checkbox-${task.index}">${task.description}</label>
+    </div>
+    <a href="#"><img src=${OptionIcon} alt="option icon" class="option-icon"></a>
+  </article>`;
+  });
+};
```

Comment:
- [ ] The todos are not being rendered. Kindly check your implementation of showing the todos from the todos array on the page. 

![image](https://user-images.githubusercontent.com/50941074/206312829-1ba38291-4703-4264-877c-c7bf86695694.png)